A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_AAF.sqf

```diff
@@ -119,13 +122,94 @@ if (_hasWs) then {
     _aa pushBack "I_A_Truck_02_aa_lxWS";
 };
 
-["vehiclesLightTanks",  _lightTanks] call _fnc_saveToTemplate;
-["vehiclesCargoTrucks", _cargoTrucks] call _fnc_saveToTemplate;
-["vehiclesAA", _aa] call _fnc_saveToTemplate;
+//If GM cdlc + extra AAF mod
+if (isClass (configFile >> "cfgVehicles" >> "gmx_aaf_m113a2dk_wdl") && _hasGM) then {
```

Comment:
use [hasAddon](https://github.com/SilenceIsFatto/A3-Antistasi-Ultimate/blob/stable/A3A/addons/ultimate/functions/Utility/fn_hasAddon.sqf)

```suggestion
private _hasAAFGM = ["gmx_aaf_m113a2dk_wdl"] call A3U_fnc_hasAddon;
if (_hasGM && _hasAAFGM) then {
```

Reply:
makes it more readable, so yes